azure-pipelines.yml

```diff
@@ -166,29 +166,18 @@ stages:
     # Publish to Build Asset Registry in order to generate the ReleaseConfigs artifact.
     - template: /eng/common/templates/job/publish-build-assets.yml
       parameters:
+        publishUsingPipelines: true
```

Comment:
?

Does this stop it from assigning to the channel during the publish to BAR step so that the promote job can do it? Do I need to port it to `master` and `release/3.0`?

Reply:
Yeah, that's what Cesar said.  Let's see what happens with the p9 build to verify.

Reply:
Based on log messages, appears to.

Reply:
I see the instruction in https://github.com/dotnet/arcade/blob/master/Documentation/CorePackages/YamlStagesPublishing.md now to do this. I missed it because I ignored the section once it started talking about `/eng/common/templates/jobs/jobs.yml` and `/eng/common/templates/job/job.yml`, which Core-Setup doesn't use. (It also doesn't explain what it does, but that probably wouldn't have stopped me from missing it during the initial implementation.)